src/__tests__/TimelineComponent.test.tsx

```diff
@@ -198,6 +199,204 @@ describe('<TimelineComponent /> R2 Tests\n', () => {
   });
 });
 
+describe('<TimelineComponent /> R3 Tests\n', ()=>{
+  const data = new Data('path/to/file', [
+    {
+      'Region': 'North America',
+      'Country': 'Central African Republic',
+      'Item Type': 'Vegetables',
+      'Sales Channel': 'Online',
+      'Order Priority': 'H',
+      'Order Date': '1/1/2010',
+      'Order ID': 506209075,
+      'Ship Date': '2/4/2010',
+      'Units Sold': 7369,
+      'Unit Price': 154.06,
+      'Unit Cost': 90.93,
+      'Total Revenue': 1135268.14,
+      'Total Cost': 670063.17,
+      'Total Profit': 465204.97,
+      'index': 4535,
+      'Order Date_num': 1262325600000
+    },
+    {
+      'Region': 'North America',
+      'Country': 'China',
+      'Item Type': 'Cereal',
+      'Sales Channel': 'Online',
+      'Order Priority': 'C',
+      'Order Date': '1/1/2010',
+      'Order ID': 863776719,
+      'Ship Date': '2/10/2010',
+      'Units Sold': 9581,
+      'Unit Price': 205.7,
+      'Unit Cost': 117.11,
+      'Total Revenue': 1970811.7,
+      'Total Cost': 1122030.91,
+      'Total Profit': 848780.79,
+      'index': 5104,
+      'Order Date_num': 1262325600000
+    },
+    {
+      'Region': 'North America',
+      'Country': 'Sweden',
+      'Item Type': 'Clothes',
+      'Sales Channel': 'Online',
+      'Order Priority': 'H',
+      'Order Date': '1/2/2010',
+      'Order ID': 907228076,
+      'Ship Date': '2/21/2010',
+      'Units Sold': 7803,
+      'Unit Price': 109.28,
+      'Unit Cost': 35.84,
+      'Total Revenue': 852711.84,
+      'Total Cost': 279659.52,
+      'Total Profit': 573052.32,
+      'index': 4193,
+      'Order Date_num': 1262412000000
+    },
+    {
+      'Region': 'North America',
+      'Country': 'Equatorial Guinea',
+      'Item Type': 'Snacks',
+      'Sales Channel': 'Offline',
+      'Order Priority': 'M',
+      'Order Date': '1/2/2010',
+      'Order ID': 335552775,
+      'Ship Date': '2/12/2010',
+      'Units Sold': 6378,
+      'Unit Price': 152.58,
+      'Unit Cost': 97.44,
+      'Total Revenue': 973155.24,
+      'Total Cost': 621472.32,
+      'Total Profit': 351682.92,
+      'index': 7005,
+      'Order Date_num': 1262412000000
+    },
+    {
+      'Region': 'North America',
+      'Country': 'Mongolia',
+      'Item Type': 'Cosmetics',
+      'Sales Channel': 'Offline',
+      'Order Priority': 'C',
+      'Order Date': '1/2/2010',
+      'Order ID': 695167052,
+      'Ship Date': '1/22/2010',
+      'Units Sold': 4234,
+      'Unit Price': 437.2,
+      'Unit Cost': 263.33,
+      'Total Revenue': 1851104.8,
+      'Total Cost': 1114939.22,
+      'Total Profit': 736165.58,
+      'index': 9127,
+      'Order Date_num': 1262412000000
+    }],
```

Comment:
Don't we already have this data in multiple places? Can we move it somewhere global so every test can use it?